src/plone/restapi/serializer/dxcontent.py

```diff
@@ -84,6 +86,24 @@ def __call__(self, version=None, include_items=True):
                 value = serializer()
                 result[json_compatible(name)] = value
 
+        # Fetch discussion registry
```

Comment:
@sneridagh maybe we should add a comment here where to find the corresponding plone.app.discussion code.

Reply:
Done.